sqldb-django-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,249 @@
+        "version": {
+            "type": "string",
+            "defaultValue": "12.0"
+        }
+    },
+    "variables": {
+        "nicName": "myVMNic",
+        "addressPrefix": "10.0.0.0/16",
+        "subnetName": "Subnet-1",
+        "subnetPrefix": "10.0.0.0/24",
+        "vmSourceImageName": "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-14_04_2_LTS-amd64-server-20150309-en-us-30GB",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressName": "myPublicIP",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "[parameters('vmDnsName')]",
+        "vmSize": "Standard_A0",
+        "virtualNetworkName": "MyVNET",
+        "sourceImageName": "[concat('/',subscription().subscriptionId,'/services/images/',variables('vmSourceImageName'))]",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+        "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]"
+    },
+    "resources": [
+
+      {
+      "apiVersion": "2014-04-01-preview",
+      "location": "[parameters('serverLocation')]",
+      "name": "[parameters('serverName')]",
+      "properties": {
+        "administratorLogin": "[parameters('administratorLogin')]",
+        "administratorLoginPassword": "[parameters('administratorLoginPassword')]",
+        "version": "[parameters('version')]"
+      },
+      "resources": [
+        {
+          "apiVersion": "2014-04-01-preview",
+          "dependsOn": [
+            "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+          ],
+          "location": "[parameters('serverLocation')]",
+          "name": "[parameters('databaseName')]",
+          "properties": {
+             "collation": "[parameters('collation')]",
+             "edition": "[parameters('edition')]",
+             "maxSizeBytes": "[parameters('maxSizeBytes')]",
+             "requestedServiceObjectiveId": "[parameters('requestedServiceObjectiveId')]"
+           },
+          "type": "databases"
+        },
+        {
+          "apiVersion": "2014-04-01-preview",
+          "dependsOn": [
+            "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+          ],
+          "location": "[parameters('serverLocation')]",
+          "name": "FirewallRule1",
+          "properties": {
+            "endIpAddress": "[parameters('firewallEndIP')]",
+            "startIpAddress": "[parameters('firewallStartIP')]"
+          },
+          "type": "firewallrules"
+        }
+      ],
+      "type": "Microsoft.Sql/servers"
+    },
+    {
+        "type": "Microsoft.Storage/storageAccounts",
+        "name": "[parameters('newStorageAccountName')]",
+        "apiVersion": "2014-12-01-preview",
+        "location": "[parameters('location')]",
+        "properties": {
+            "accountType": "[variables('storageAccountType')]"
+        }
+    }, {
+        "apiVersion": "2014-12-01-preview",
```

Comment:
Please move to 2015-05-01-preview API version header for storage, network and compute resources